src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -100,6 +102,18 @@ private FirebaseUserManager(Builder builder) {
     this.httpClient = new AuthHttpClient(jsonFactory, builder.requestFactory);
   }
 
+  private String getIdToolkitUrl(String projectId, String version) {
+    String maybeEmulator = getFirebaseAuth();
+    if (maybeEmulator != null) {
+      return String.format(ID_TOOLKIT_URL_EMULATOR, maybeEmulator, version, projectId);
+    }
+    return String.format(ID_TOOLKIT_URL, version, projectId);
+  }
+
+  private String getFirebaseAuth() {
```

Comment:
Give this method a more appropriate name? How about `getAuthEmulatorHost` or `getEmulatorHost`, since the current context is already within Firebase Auth :)